project.godot

```diff
@@ -246,8 +246,6 @@ translations=PoolStringArray( "res://locale/en.po", "res://locale/fr.po", "res:/
 [logging]
 
 file_logging/enable_file_logging=true
-file_logging/log_path="user://logs/log.txt"
-file_logging/max_log_files=8
```

Comment:
There's still unintended changes here. https://github.com/Revolutionary-Games/Thrive/pull/2052#discussion_r563673791

Reply:
yes I'm sorry understanding problem here

Reply:
may be fix now